iron-overlay-behavior.html

```diff
@@ -265,6 +280,14 @@
     },
 
     _withBackdropChanged: function() {
+      // If tabindex is already set, no need to override it.
+      if (this.withBackdrop && !this.hasAttribute('tabindex')) {
+        this.setAttribute('tabindex', '-1');
+        this.__shouldRemoveTabIndex = true;
+      } else if (this.__shouldRemoveTabIndex) {
+        this.removeAttribute('tabindex');
+        this.__shouldRemoveTabIndex = false;
+      }
```

Comment:
Since we only need `tabindex` for focus purposes, any value is good. If someone wants to set a `tabindex=0` while `with-backdrop`, `iron-overlay-behavior` should allow it and not override it.